workspaces/Cargo.toml

```diff
@@ -10,9 +10,11 @@ Library for automating workflows and testing NEAR smart contracts.
 
 [dependencies]
 async-trait = "0.1"
+async-process = "1.3.0"
```

Comment:
might be a good idea to make this optional under the `unstable` flag as well, but not a big deal

Reply:
Good catch! Yeah, it probably should be optional and pulled in with this feature